operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -1019,28 +1027,30 @@ local utils = import 'mixin-utils/utils.libsonnet';
     );
     local labelBreakouts = '%s' % std.join(', ', labels);
     local histogramLabelBreakouts = '%s' % std.join(', ', ['le'] + labels);
+
+    local percentileTargets = [
+      {
+        expr: histogramExprTmpl % [percentile, metricName, selector, histogramLabelBreakouts, multiplier],
+        format: 'time_series',
+        legendFormat: '%sth Percentile: {{ %s }}' % [std.lstripChars(percentile, '0.'), labelBreakouts],
+        refId: 'A',
+      }
+      for percentile in percentiles
+    ];
+    local averageTargets = [
+      {
+        expr: averageExprTmpl % [metricName, selector, labelBreakouts, multiplier, metricName, selector, labelBreakouts],
+        format: 'time_series',
+        legendFormat: 'Average: {{ %s }}' % [labelBreakouts],
```

Comment:
[nit] If we're only showing one percentile or just the average on a panel, the `%sth Percentile: ` / `Average: ` prefix is superfluous

Reply:
I figured I would leave it for the option as the func here can still be used show them both on the same chart if desired.